### DISCUSSION TITLE
Welcome to SwiftHub Discussions!

### DISCUSSION THREAD

**Discussion by khoren93** [khoren93/SwiftHub#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EdwardXHLam** [comment]
Hi, 
    Here is Edward, i would like to further study RxSwift, SwiftHub is a very good project for me , but I have spent a lot of time, still could not successfully run the project, I have followed your tips, even the last command "bundle exec fastlane update" is perfectly run, but after successfully "pod install", it will show error description, showing below.
    Kindly could you help?
Sincerely,
Edward


firebase-ios-sdk
invalidManifestFormat("truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)<unknown>:0: error: file \'/var/folders/dc/rxkmz_t50g90wcgl6hx94v4r0000gn/C/clang/ModuleCache/H293QAI19N7H/Darwin-1IVCWVLR6MT9T.pcm\' is not a valid precompiled module file\n<unknown>:0: error: cannot load underlying module for \'Darwin\'\ntruncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)truncated or malformed object (offset field plus size field of section 1 in LC_SEGMENT_64 command 0 extends past the end of the file)", diagnosticFile: Optional(<AbsolutePath:"/Users/yz-lxh/Library/Developer/Xcode/DerivedData/SwiftHub-frswgbjahqdsbjceodsjzsosyhqs/SourcePackages/ManifestLoading/firebase-ios-sdk.dia">)) in https://github.com/firebase/firebase-ios-sdk.git

Fetching from https://github.com/firebase/firebase-ios-sdk.git
Fetching from https://github.com/firebase/firebase-ios-sdk.git


Resolving Package Graph Failed    2022/4/14, 11:33 AM    492.6 seconds
